.openpublishing.redirection.json

```diff
@@ -185,6 +185,11 @@
             "redirect_url":"/cli/azure/use-azure-cli-successfully",
             "redirect_document_id": false
          },
+         {
+            "source_path":"docs-ref-conceptual/authenticate-azure-cli-web-account-manager.md",
+            "redirect_url":"/cli/azure/authenticate-azure-cli-interactively",
+            "redirect_document_id": false
+         },
```

Comment:
Since `authenticate-azure-cli-web-account-manager` is no longer used, it's better for us to redirect https://go.microsoft.com/fwlink/?linkid=2271136 (currently pointing to https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-web-account-manager) to https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-interactively. @dcaro